tests/rust/required_fields_should_not_be_option_generated.rs

```diff
@@ -0,0 +1,440 @@
+// automatically generated by the FlatBuffers compiler, do not modify
```

Comment:
Why create a new directory instead of using the existing directories in our test codebase?